src/components/contacts/ContactRow.js

```diff
@@ -111,7 +106,6 @@ const ContactRow = (
         const name = await fetchReverseRecord(address);
         if (name !== ensName) {
           setENSName(name);
-          onAddOrUpdateContacts(address, nickname, color, network, name);
```

Comment:
i think we want this, this would allow the ens to be updated when this is mounted